controller/accounts.js

```diff
@@ -127,45 +254,54 @@ router.post('/login', function(req, res){
 // /home post project
 router.post('/home', function(req, res){
 // execute jira-miner target to point at the source
-  child = exec('jira-miner populate "project in (' + req.body.project + ')"',{maxBuffer: 1024 * 20000}, function (error, stdout, stderr) {
-    console.log(stdout);
-    if(stdout.indexOf('Updated and stored collection') >= 0 ){
-      message = 'Connected to '+ req.body.project +' project';
-    }else{
-      message = 'Failed to connect to '+req.body.project +' project' + stderr;
-    }
-    res.render('home',{title: 'Kujira Home', message: message});
-  });
+  if(!sess || !sess.username){
+    res.redirect('/');
+  } else {
+    child = exec('jira-miner populate "project in (' + req.body.project + ')"', {maxBuffer: 1024 * 20000}, function (error, stdout, stderr) {
+      console.log(stdout);
+      if (stdout.indexOf('Updated and stored collection') >= 0) {
+        message = 'Connected to ' + req.body.project + ' project';
+      } else {
+        message = 'Failed to connect to ' + req.body.project + ' project' + stderr;
+      }
+      res.render('home', {title: 'Kujira Home', message: message});
+    });
+  }
 });
 
 // /home post project
 router.post('/query', function(req, res){
 // execute jira-miner target to point at the source
-  if (searchString.indexOf(req.body.value) >= 0){
```

Comment:
probably just need one conditional here, ie if no searchstring then build one, no need for the if